lock/etcd.go

```diff
@@ -25,19 +27,29 @@ import (
 
 const (
 	keyPrefix       = "coreos.com/updateengine/rebootlock"
-	holdersPrefix   = keyPrefix + "/holders"
 	SemaphorePrefix = keyPrefix + "/semaphore"
+	groupPath       = "groups"
+	semaphorePath   = "semaphore"
```

Comment:
These don't seem appropriately named to me, they're path components, but a single component of a path isn't generally referred to as a path.

Perhaps s/Path/Branch/g?

And make SemaphorePrefix = keyPrefix + semaphoreBranch?